docs/gateway-api-compatibility.md

```diff
@@ -71,7 +71,13 @@ Fields:
 	* `name` - supported.
 	* `supportedKinds` - not supported.
 	* `attachedRoutes` - supported.
-	* `conditions` - partially supported.
+	* `conditions` - Supported (Condition/Status/Reason):
```

Comment:
the indentation looks odd in the lines below

Reply:
fixed